src/main/java/in/twizmwaz/cardinal/module/modules/titleRespawn/TitleRespawnBuilder.java

```diff
@@ -0,0 +1,32 @@
+package in.twizmwaz.cardinal.module.modules.titleRespawn;
+
+import in.twizmwaz.cardinal.match.Match;
+import in.twizmwaz.cardinal.module.BuilderData;
+import in.twizmwaz.cardinal.module.ModuleBuilder;
+import in.twizmwaz.cardinal.module.ModuleCollection;
+import in.twizmwaz.cardinal.module.ModuleLoadTime;
+import in.twizmwaz.cardinal.util.StringUtils;
+import org.jdom2.Element;
+
+@BuilderData(load = ModuleLoadTime.NORMAL)
+public class TitleRespawnBuilder implements ModuleBuilder {
+
+    @Override
+    public ModuleCollection load(Match match) {
+        Element head = match.getDocument().getRootElement().getChild("respawn");
+
+        int delay = StringUtils.timeStringToSeconds(head.getAttributeValue("delay", "2s"));
+        if (delay < 1) {
+            delay = 1;
+        }
+
+        boolean auto = Boolean.parseBoolean(head.getAttributeValue("auto", "false"));
```

Comment:
You may want to use NumUtils#parseBoolean, as it allows the XML to use `on` and `off` in addition to `true` and `false`.